### PR TITLE
multiclass-classification

### PR DESCRIPTION
**Pull request by evanyeohboonkhai**
When you added a new program / programs , put a cross on the  OS that you have test it on  
(Example: [ X ] Windows  )

[ X] Windows  
[  ] Linux Ubuntu  
[  ] Mac  
[  ] Others  (State here -> xxx )  

Assign a reviewer to review your code.

### CODE REVIEW

**Comment by kenghooi-teoh** [comment]
@evanyeohboonkhai , good progress but there's a few updates required:
1. provide evaluation stats for train set as well
2. create a same file with incomplete/blank code as an exercise. the file you created is supposed to be a solution file
3. consistency in the styling e.g. spacing afer/before equal signs, redudant blank lines etc.

File: dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+import java.util.List;
+
+
+//Dataset
+//https://www.kaggle.com/zhangjuefei/birds-bones-and-living-habits
+public class MulticlassExcercise {
+    static int seed =123;
+    static int numInput=10;
+    static int numClass=6;
+    static int epoch=9000;
```
Comment (codenamewei) [inline dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java:47]: 9000 epochs???? Is this necessary

File: dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+//Dataset
+//https://www.kaggle.com/zhangjuefei/birds-bones-and-living-habits
+public class MulticlassExcercise {
```
Comment (codenamewei) [inline dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java:43]: CamelCase for java programming... Also pay more attention on the details of Typo.

Basically each word first letter should be capital .

File: dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+    static int seed =123;
+    static int numInput=10;
+    static int numClass=6;
+    static int epoch=9000;
+    static int batch_size=64;
+    static double splitratio =0.7;
```
Comment (codenamewei) [inline dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java:49]: for variable name camel case, it's first word small letter, and the rest Camel Case

```
splitRatio
batchSize
```

File: dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+//        //Create Iterator and shuffle the data
+        //Error due to zero data is obtain
+        DataSet fulldataset = dataSetIterator.next();
+        fulldataset.shuffle();
+//
+//        //Input split ratio
+        SplitTestAndTrain testAndTrain = fulldataset.splitTestAndTrain(splitratio);
+//
+          //Get train and test dataset
+        DataSet trainData =testAndTrain.getTrain();
```
Comment (codenamewei) [inline dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java:106]: DataSet trainData = testAndTrain.getTrain();

space before and after the operator

File: dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+//Dataset
+//https://www.kaggle.com/zhangjuefei/birds-bones-and-living-habits
```
Comment (codenamewei) [inline dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java:42]: add author name 

so we know who to look back to for this program

Example
```
/***
 * Loader status for project loading
 *
 * @author Chiawei Lim
 */
@Slf4j
public enum LoaderStatus
{
    ERROR,
    LOADING,
    LOADED, //projectloader will have this status once create new project
    DID_NOT_INITIATED, //default value when ProjectLoader created from database
}
```

**Review by codenamewei — COMMENTED** [review]
Check these up :)

File: dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java [hunk 6]
```diff
@@ -0,0 +1,188 @@
+package ai.certifai.training.classification.Multiclass_boon_khai;
```
Comment (kenghooi-teoh) [inline dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java:1]: package name incorrect

File: dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+
+
+//Dataset
+//https://www.kaggle.com/zhangjuefei/birds-bones-and-living-habits
+public class MulticlassExcercise {
+    static int seed =123;
+    static int numInput=10;
+    static int numClass=6;
+    static int epoch=9000;
+    static int batch_size=64;
```
Comment (kenghooi-teoh) [inline dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java:48]: batch_size is not used?

File: dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+        model.setListeners(new ScoreIterationListener(10));
+        model.init();
+
+        //Evaluator
+        UIServer server = UIServer.getInstance();
+        StatsStorage storage = new InMemoryStatsStorage();
+        server.attach(storage);
+
+        //Training
+        for (int i = 0; i < epoch; ++i) {
```
Comment (kenghooi-teoh) [inline dl4j-labs/src/main/java/ai/certifai/training/classification/MulticlassExcercise.java:138]: this for loop can reduced to a one-liner `model.fit(trainData, epoch)` if you are not doing anything else in the for loop

**Comment by codenamewei** [comment]
@evanyeohboonkhai adding new program require updating the readme too 
https://github.com/CertifaiAI/TrainingLabs/blob/master/README.md

**Comment by evanyeohboonkhai** [comment]
@kenghooi-teoh changes made , please review again

File: dl4j-labs/src/main/java/ai/certifai/solution/classification/MultiClassExercise.java [hunk 9]
```diff
@@ -0,0 +1,229 @@
...
+ * Loader status for project loading
+ *
+ * @author BoonKhai Yeoh
+ */
+
+//Dataset
+//https://www.kaggle.com/zhangjuefei/birds-bones-and-living-habits
+public class MultiClassExercise {
+    static int seed =123;
+    static int numInput=10;
```
Comment (kenghooi-teoh) [inline dl4j-labs/src/main/java/ai/certifai/solution/classification/MultiClassExercise.java:49]: @evanyeohboonkhai fix spacing before/after equal sign

File: dl4j-labs/src/main/java/ai/certifai/solution/classification/MultiClassExercise.java [hunk 10]
```diff
@@ -0,0 +1,229 @@
...
+    public static MultiLayerConfiguration getConfig(int numInputs, int numOutputs, double learningRate) {
+
+        MultiLayerConfiguration config = new NeuralNetConfiguration.Builder()
+                .seed(seed)
+                .weightInit(WeightInit.XAVIER)
+                .updater(new Nesterovs(learningRate, Nesterovs.DEFAULT_NESTEROV_MOMENTUM))
+                .l2(0.001)
+                .list()
+                .layer(0, new DenseLayer.Builder()
+                        .nIn(numInputs)
```
Comment (kenghooi-teoh) [inline dl4j-labs/src/main/java/ai/certifai/solution/classification/MultiClassExercise.java:200]: use smaller model

File: dl4j-labs/src/main/java/ai/certifai/solution/classification/MultiClassExercise.java [hunk 11]
```diff
@@ -0,0 +1,229 @@
...
+
+/***
+ * Loader status for project loading
+ *
+ * @author BoonKhai Yeoh
+ */
+
+//Dataset
+//https://www.kaggle.com/zhangjuefei/birds-bones-and-living-habits
+public class MultiClassExercise {
```
Comment (kenghooi-teoh) [inline dl4j-labs/src/main/java/ai/certifai/solution/classification/MultiClassExercise.java:47]: rename to BirdClassification

File: dl4j-labs/src/main/java/ai/certifai/solution/classification/MultiClassExercise.java [hunk 12]
```diff
@@ -0,0 +1,229 @@
...
+
+        //Create iterator from process data
+        CollectionRecordReader collectionRR = new CollectionRecordReader(processData);
+
+        //Input batch size , label index , and number of label
+        DataSetIterator dataSetIterator = new RecordReaderDataSetIterator(collectionRR, processData.size(),-1,6);
+
+//        //Create Iterator and shuffle the dat
+        DataSet fullDataset = dataSetIterator.next();
+        fullDataset.shuffle();
```
Comment (kenghooi-teoh) [inline dl4j-labs/src/main/java/ai/certifai/solution/classification/MultiClassExercise.java:115]: set seed
